### DISCUSSION TITLE
PAT Basic 1023. 组个最小数 (20) (C语言实现) | PAT Solution in C

### DISCUSSION THREAD

**Discussion by xlucn** [xlucn/PAT#73] [General]
https://oliverlew.github.io/PAT/Basic/1023.html 

 题目

**Comment by Huuuuu123** [comment]
怎么可以这么厉害，崇拜了，我大一刚学，我想问问大佬一开始就学得这么厉害嘛

**Comment by Huuuuu123** [comment]
每答对一道，就回来看看你的解法，大部分都会把我吓到。

**Comment by xlucn** [comment]
> @Huuuuu123
> 怎么可以这么厉害，崇拜了，我大一刚学，我想问问大佬一开始就学得这么厉害嘛

哈哈谢谢夸奖，不过我都不是专业的，还是不能和科班的学生们比，我这都是用时间熬出来的
